proxy/FetchSM.cc

```diff
@@ -26,6 +26,7 @@
 #include <stdio.h>
 #include "HTTP.h"
 #include "PluginVC.h"
+#include <api/ts/ts.h> // Ugly, but we need a bunch of the public APIs here ... :-/
```

Comment:
Can you make this "api/ts/ts.h" for consistency with the other place you do this?